backend/src/polling_station/structs.rs

```diff
@@ -50,6 +50,7 @@ pub struct PollingStation {
     pub house_number_addition: Option<String>,
     pub postal_code: String,
     pub locality: String,
+    pub status: PollingStationStatus,
```

Comment:
Why extend this api endpoint with a status field? We already have a separate endpoint to return the status for each polling station, and that endpoint is more suitable for refreshing often. This endpoint currently returns very static data and I don't see a good reason to add the relatively dynamic status field to that.

Reply:
It made the frontend implementation a lot easier. I hadn't considered the staticness of the struct as it was.

Reply:
Thanks for removing it! I think the frontend implementation wasn't too complicated after all? And this way, reloading the statuses (#336) should be a bit more lightweight.